README.md

```diff
@@ -243,6 +244,29 @@ Generatr container. So
 is needed to expose the container's port 8080 to the host (web browser). In the example above, the
 `-p 8080:8080` argument tells Docker to bind the local machine / host's port 8080 to the container's port 8080.
 
+#### Simplify using Docker-Compose
+Using Docker-Compose we can simplify the process.
```

Comment:
Docker Compose is written without the hyphen. Also, a link to the Docker Compose site would be nice.

```suggestion
#### Simplify using Docker Compose
Using [Docker Compose](https://docs.docker.com/compose) we can simplify the process.
```